client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
Status Change dialog
![Screenshot of the Status Change dialog showing 14px font size](https://github.com/MESH-Research/Pilcrow/assets/2668987/c4acde58-bc4c-4bcf-a822-b61a895f795d)

Reply:
These now appear as 16px.